checkout-ui-custom/src/controller/DeliverController.js

```diff
@@ -63,10 +64,18 @@ const DeliverController = (() => {
     preparePhoneField(`#${EDIT_FORM_RECEIVER_PHONE_ID}`);
   };
 
+  const RenderAddAddressManual = async () => {
+    document.querySelector('#manual-address-section').innerHTML = AddAddressManual();
```

Comment:
Have you tested the validation on your form field: you need to prepare the Field